linz_logger/logger.py

```diff
@@ -88,6 +90,8 @@ def add_default_keys(current_logger, method_name: str, event_dict: dict):
     # Standard keys that need to be added
     event_dict["v"] = 1
     event_dict["pid"] = pid
+    event_dict["hostname"] = node()
```

Comment:
you should do this once, the hostname should never change mid run?, see how `pid` is handled.